### DISCUSSION TITLE
Metric for identifying the http requests and the response returned for the request

### DISCUSSION THREAD

**Discussion by prs9728** [open-telemetry/opentelemetry-collector#9160] [General]
Hi, I am looking for a metric  or any way to identify the no.of http requests received by the collector and the response for each of the request (status code). Could any one point me to a documentation or how this information can be retrived

Thanks

**Comment by atoulme** [comment]
See https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md
We set up the otelhttp instrumentation on the http.Server we create via confighttp.
